apps/prairielearn/src/lib/courseUtil.js

```diff
@@ -31,7 +31,7 @@ module.exports.getCommitHashAsync = promisify(getCommitHash);
  * will also add the `commit_hash` property to the given course object.
  *
  * @param {Object} course
- * @param {(err?: Error, commitHash?: string) => void} callback
+ * @param {(err?: Error, commitHash: string) => void} callback
```

Comment:
`instructorFileEditor` is the last place where these callback-based functions are used; we should be able to remove them very soon!

Reply:
Yay!